arviz/stats/diagnostics.py

```diff
@@ -13,10 +13,11 @@
     autocov as _autocov,
     not_valid as _not_valid,
     wrap_xarray_ufunc as _wrap_xarray_ufunc,
+    stats_variance_2d as svar,
 )
 from ..data import convert_to_dataset
-from ..utils import _var_names
-
+from ..utils import _var_names, conditional_jit, conditional_vect
+from .. import Numba
```

Comment:
I _think_ that the cyclic import problem comes from this line. This line is in both diagnostics.py and stats.py, thus, it would explain why the cyclic import warnings finish always in one of these two files, and it looks like it is importing from _all_ ArviZ, hence the cycle. I don't really know how to import things from the __init__ file from within the same library though.

Reply:
init too imports stats, plots and data. Hence the cyclic import I guess.